docs/sources/auth/ldap.md

```diff
@@ -92,6 +92,14 @@ member_of = "memberOf"
 email =  "email"
 ```
 
+### Using environment variables
+
+You can interpolate variables in the TOML config from environment variables. For instance, you could externalize your `bind_password` that way:
```

Comment:
```suggestion
You can interpolate variables in the TOML config from environment variables. For instance, you could externalize your `bind_password` like this:
```

Reply:
Hi oddli
This my docker file looks like 
ARG BASE_IMAGE=alpine:3.12
FROM ${BASE_IMAGE}

ENV LDAP_ADMIN_PASSWORD

and I pass through  Docker run , But it never changed  in ldap config file ?
Is the any documentation How we can execute or sample docker file ?. I have been struggling from 3 days  :(